persistence.go

```diff
@@ -93,7 +86,7 @@ func (db sqlitePersistence) SaveChat(chat Chat) error {
 	defer stmt.Close()
 
 	_, err = stmt.Exec(
-		chatID,
+		chat.ID,
```

Comment:
Don't we want to do some kind of argument validation on method entry?

Reply:
probably, maybe though I will leave for a separate PR, we might want to think about what kind of validation and where exactly we want to have it